kyo-data/shared/src/main/scala/kyo/Duration.scala

```diff
@@ -58,7 +59,7 @@ object Duration:
       * @return
       *   A Duration instance
       */
-    inline def fromNanos(value: Long): Duration =
```

Comment:
Something that was quite annoying while working on this change were assertion errors reporting inlined trees from `Duration` operations. It becomes a large and confusing error message. Since these methods don't take functions, the benefit of inlining them is low so I've decided to remove the modifier

Reply:
Sounds good, I think this is the right move. I've noticed this issue with `loop` as well. If you incorrectly return a non-loop (Done/Continue) value the expanded tree is massive.